prosessering-core/src/main/kotlin/no/nav/familie/prosessering/internal/TaskScheduler.kt

```diff
@@ -54,7 +54,7 @@ class TaskScheduler(private val taskMaintenanceService: TaskMaintenanceService)
         if (isOptimisticLocking(e)) {
             logger.warn("OptimisticLockingFailureException metode=$metode")
         } else {
-            logger.error("Feilet metode=$metode. Se secure logs")
+            logger.warn("Feilet metode=$metode. Se secure logs")
```

Comment:
Er ikke dette litt skummelt? For denne tar jo inn alle typer exceptions, ikke bare optimisticLock ? 

Hvorfor slår ikke feilene våre ut på den if-sjekken ett hakk opp?

Reply:
Jeg var litt for rask her.. Og gjorde feil.
Feilet er ju faktiskt en PessimisticLockingFailureException

Reply:
Sjekke for ConcurrencyFailureException i stedet ?